.buildkite/bk.integration.pipeline.yml

```
@@ -133,6 +133,8 @@ steps:
           - fleet-airgapped-privileged
 
       - label: "arm:sudo: {{matrix}}"
+        # https://github.com/elastic/ingest-dev/issues/4610
+        skip: true
```

While skip supports `true|false` it also can support the comment so that we could use this instead 
```suggestion
        skip: Due to https://github.com/elastic/ingest-dev#4610
```

So the reason will be show in the UI.